cmd/serve/sftp/sftp.go

```diff
@@ -101,6 +115,22 @@ sftp backend, but it may not be with other SFTP clients.
 		}
 		cmd.Run(false, true, command, func() error {
 			s := newServer(context.Background(), f, &Opt)
+			if Opt.Stdio {
+				if terminal.IsTerminal(int(os.Stdout.Fd())) {
+					return errors.New("Refusing to run SFTP server directly on a terminal. Please let sshd start rclone, by connecting with sftp or sshfs.")
+				}
+				sshChannel := &StdioChannel{
+					stdio: restic.NewStdioConn(os.Stdin, os.Stdout),
+					stderr: os.Stderr,
+				}
+				handlers := newVFSHandler(vfs.New(f, &vfsflags.Opt))
+				server := sftp.NewRequestServer(sshChannel, handlers)
```

Comment:
`NewRequestServer` takes an `io.ReadWriteCloser` which needs a Read/Write/Close method. So I think your definition of sshChannel can be simplified.  Unless it is doing interface upgrades...?

Reply:
Done.